### PR TITLE
Bug 1534278 –  Remove Extra borders on Hero cards on Dark theme

### PR DESCRIPTION
**Pull request by gvn**
Repro: https://bugzilla.mozilla.org/show_bug.cgi?id=1534278#c0

### CODE REVIEW

File: content-src/styles/_mixins.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 }
 
 @mixin dark-theme-only {
-  [lwt-newtab-brighttext]:not(.force-light-theme) & {
```
Comment (gvn) [inline content-src/styles/_mixins.scss]: `force-light-theme` appears to be deprecated.

Comment (punamdahiya) [reply content-src/styles/_mixins.scss]: yes with [Enable Dark Mode](https://github.com/mozilla/activity-stream/commit/2225e93564508e4ace4e69f2cd2b735fb3c5ee59#diff-3f95337718772ca7ed0b5acae1358dd0) , we are no longer using force-light-theme and should remove it from rest of the places where as its used.

https://searchfox.org/mozilla-central/search?q=force-light-theme&path=
https://github.com/mozilla/activity-stream/commit/8f3f2f6a4660d8bf557e9823ec2d7e78415775b4

Comment (punamdahiya) [reply content-src/styles/_mixins.scss]: @gvn as this is follow up of enabling dark mode,  can we do this cleanup as part of this PR?

File: content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss [hunk 2]
```diff
@@ -61,18 +56,17 @@ $card-header-in-hero-line-height: 20;
...
-    @include ds-border-bottom;
     @include dark-theme-only {
       color: $grey-30;
     }
 
     color: $grey-50;
     display: block;
     margin: 12px 0 16px;
-    padding: 16px 0;
+    padding: 16px 0 0;
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss]: nit: we can switch to padding-top here

File: content-src/styles/_mixins.scss (hunk 1 above)
Comment (gvn) [reply content-src/styles/_mixins.scss]: I would prefer to do that as a separate ticket, but I will file it!

**Comment by gvn** [comment]
My guess is we can probably get rid of the dark mode message component as well as part of the force-light-theme cleanup.

File: content-src/styles/_mixins.scss (hunk 1 above)
Comment (punamdahiya) [reply content-src/styles/_mixins.scss]: sounds good, thanks!

**Pull request merged**